app/org/sagebionetworks/bridge/play/controllers/AuthenticationController.java

```diff
@@ -100,6 +102,7 @@ private Result signInWithRetry(final int retryCounter) throws Exception {
         final Study study = getStudyOrThrowException(json);
         try {
             session = authenticationService.signIn(study, signIn);
+            verifySupportedVersionOrThrowException(study, session);
```

Comment:
Would like to keep the naming w/o the "min" since the exception it throws is "unsupported".